packages/off-chain/inputs/dummyInput.json

```diff
@@ -0,0 +1,10 @@
+{
+  "data": {
+    "param1": 10,
+    "param2": 20
+  },
+  "function": {
+      "params": ["param1", "param2"],
+      "code": "return param1 + param2"
+  }
+}
```

Comment:
I get that this is just for testing purposes, but just to make sure: later on the actual contribution data and the definition of the IE function should be two separate inputs.

Reply:
Yes, the parameters and the function definition should be in two separate properties.

The reason being, `eval` runs the code directly, so we won't be able to pass any parameters. Also, using `eval` is a security risk as any code will run in the global context and can be dangerous (though, no high risk processing is done here).

Whereas `Function` class, accepts parameters and creates an instance of function, which we can invoke anytime.

Resources:
[Function()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/Function#:~:text=Description-,Function()%20constructor,performance%20issues%20as%20eval()%20.)
[eval](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval)
[eval vs Function()](https://www.educative.io/answers/eval-vs-function-in-javascript)